docs/tutorial/simple-transformations/index.md

```diff
@@ -0,0 +1,115 @@
+The QOF register for diabetes should contain all patients who, on 31st March 2024:
+
+* were at least 17 years old,
+* were alive,
+* were registered with a GP practice, and
+* had an unresolved diabetest diagnosis.
+
+To start with, we'll show you how we can transform the `patients` table to find the patients who were at least 17 years old and alive on 31st March 2024.
+
+These are very common things to want to know, so ehrQL gives us some helper functions: `patients.age_on` and `patients.was_alive_on`.
```

Comment:
We actually went with `is_alive_on` in the end:
https://docs.opensafely.org/ehrql/reference/schemas/core/#patients.is_alive_on

The thinking was that the benefits of syntactic consistency (we use the `is_` lots elsewhere) outweighed the naturalness of the tense.

I don't think it's too late to change this if you have very strong feelings (I don't think anyone's used the new functions yet) but we should have a discussion if so.